submissions/kasionio/oop-exercise/index.js

```diff
@@ -0,0 +1,70 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/kasionio/a-tiny-JS-world/tree/gh-pages
+   Web app: https://kasionio.github.io/a-tiny-JS-world
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(species, name, gender, legs, hands, saying) {
+  this.species = species,
+  this.name = name,
+  this.gender = gender,
+  this.legs = legs,
+  this.hands = hands,
```

Comment:
Why commas?

Reply:
My bad. Accidentally It comes from the objects